tox.ini

```diff
@@ -19,6 +19,6 @@ commands =
 setenv =
     PYTHONPATH = {toxinidir}
 deps =
-    .[test]
+    .[all]
```

Comment:
Thanks for finding the bug! I made this change locally and it fixes `make build` for me